packages/mobile/src/components/image/UserImage.tsx

```diff
@@ -19,12 +19,20 @@ type UseUserImageOptions = {
 export const useUserImage = ({ user, size }: UseUserImageOptions) => {
   const cid = user ? user.profile_picture_sizes || user.profile_picture : null
 
-  return useContentNodeImage({
+  const contentNodeImage = useContentNodeImage({
     cid,
     size,
     user,
     fallbackImageSource: profilePicEmpty
   })
+
+  if (user.updatedProfilePicture) {
```

Comment:
Would we want to add this to the core image hook or just a mobile only thing for now?

Reply:
It's handled separately on web for now, so this at least brings parity. Would be good to move deeper into common though